optimism/src/lookup.rs

```diff
@@ -201,4 +206,28 @@ impl<F: Field> LookupTable<F> {
                 .collect(),
         }
     }
+
+    fn _table_pad() -> Self {
+        Self {
+            _table: (1..=136)
```

Comment:
This `136` probably needs to be a constant (with a comment?), looks very magic :) Also the `0` value we explicitly don't need because zero paddings are zero sized?

Reply:
Oh, it should definitely be a constant. Actually `RATE_IN_BYTES`, I just forgot to import it and use it! 

And the fact that we don't include the zero entry is that the lookups for the padding are only triggered when the padding is at least 1 byte. So there's no need to include the trivial and unused entry when there's no padding to be done (0 bytes).